src/ape/types/__init__.py

```diff
@@ -264,6 +264,23 @@ def get(self, item: str, default: Optional[Any] = None) -> Any:
         return self.event_arguments.get(item, default)
 
 
+class ContractLogContainer(list):
+    __test__ = False
+
+    def filter(self, event, **kwargs) -> List[ContractLog]:
+        found_events = []
+        for log in self:
+            if log.event_name == event.name:
+                found = True
+                for k, v in kwargs.items():
+                    if log.event_arguments.get(k) != v:
+                        found = False
+                        break
+                if found:
```

Comment:
This has a slight issue in that it would incorrectly match non-existent attributes if the value is `None`, e.g. `.filter(foo=None)`.

A fairly common patter to address this is something like
```python
_NONE = object() # unique object to detect non-existent entries, somewhere on the module level
match = all(v == log.event_arguments.get(k, _NONE) for k, v in kwargs.items())
if match:
  found_events.append(log)
```

Reply:
changed.